Dockerfile

```diff
@@ -27,14 +26,8 @@ RUN curl -sSL https://github.com/resin-io/qemu/commit/db186a3f83454268c43fc793a4
 # - resulting binaries are located in "/usr/local/bin/qemu-*"
 ARG TARGET_ARCH=arm-linux-user,aarch64-linux-user,ppc64le-linux-user,riscv64-linux-user
 RUN mkdir build \
- && cd build \
- && ../configure --static --target-list=$TARGET_ARCH  \
- && make -j $(nproc) install
+    && cd build \
+    && ../configure --static --target-list=$TARGET_ARCH  \
+    && make -j $(nproc) install
 
-FROM busybox
-COPY --from=qemu /usr/local/bin/qemu-arm /qemu-arm
-COPY --from=qemu /usr/local/bin/qemu-aarch64 /qemu-aarch64
-COPY --from=qemu /usr/local/bin/qemu-ppc64le /qemu-ppc64le
-COPY --from=qemu /usr/local/bin/qemu-riscv64 /qemu-riscv64
 ADD register.sh /register.sh
-CMD ["/register.sh", "--reset"]
```

Comment:
This would break the usage of this image if there is no `CMD` any more.

Reply:
Yeah, I think that is not necessary for users, for example I can't using `busybox` in my project. I need debian, or alpine, or smthn else. Therefore `CMD` operation should using in `user-implementation` as I documented in README.md.

Reply:
I think about that a bit more and I has concluded that it can useful for tests. I return it back